syntax/yang.vim

```diff
@@ -6,92 +6,94 @@
 " Maintainer: 	Matt Parker <mparker@computer.org>
```

Comment:
Why don't we just delete `Version` and `Last Change` (that's what git is for). And I suggest changing `Maintainer` to `Author`.

Reply:
Fine by me - ok for you to take care of that outside of this merge request?